src/docs/user/references/JobPlannerServiceReference.adoc

```diff
@@ -49,6 +52,28 @@ Then, there are 3 ways to visualize the recurrence on the Execution Planning pan
 If a calendar has no association, you can still select it and see its recurrence on the Execution Planning panel.
 If nothing is displayed, it means the selected Calendar Definitions have no recurrence on the selected period.
 
+You can also use the Gantt chart to visualize the executions:
+
+image::calendar_planning_gantt.png[align="center"]
+
+In addition to the Execution Planning panel, the Gantt chart allows you to see the past jobs. You can see information such as their status at the Gantt generation time, statistics on their execution time, their submission time... You can see them in the tooltips, when passing your mouse over a specific job.
+
+The planned jobs (in orange) are on the row just above the actual jobs so that you can easily compare what was planned and what actually happened. You can use the legend button for a better understanding of the colors.
+
+image::calendar_planning_gantt_legend.png[align="center"]
+
+If a job has been PENDING before starting, depending on the scale you have selected, you can see a yellow part on the displayed job. This corresponds to the time the job was PENDING and helps to highlight a resource problem.
+
+The length of the displayed planned jobs can be :
+
+    - the expected execution time given by users in the workflow with the Generic Information : `JOB_EXEC_TIME`
+    - if the workflow doesn't have any `JOB_EXEC_TIME` Generic Information, it will be the average execution time
```

Comment:
Complete with: "it will be the average execution time computed from the past association executions"